modules/sdk-coin-eth/src/lib/utils.ts

```diff
@@ -455,7 +473,7 @@ export function classifyTransaction(data: string): TransactionType {
  */
 const transactionTypesMap = {
   [walletInitializationFirstBytes]: TransactionType.WalletInitialization,
-  [v1WalletInitializationFirstBytes]: TransactionType.WalletInitialization,
+  [v1CreateWalletMethodId]: TransactionType.WalletInitialization,
```

Comment:
What's the full impact of changing this? What's the downstream affect of changes users will have to make?

Reply:
Actually this statement was added to support v1 wallet initialization for polygon which is not yet available to customers. But this needs to be changed because there was some issue in the logic. So I don't think the customers or anybody else would be impacted by changing this line as v1 wallet init logic is not present in sdk for any other ethlike coin at this point anyways

`[v1WalletInitializationFirstBytes]: TransactionType.WalletInitialization,`